reference/6/Microsoft.PowerShell.Core/About/about_Split.md

```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
 
 The SimpleMatch options are:
 
-* **SimpleMatch**: Use simple string comparison when evaluating the
+- **SimpleMatch**: Use simple string comparison when evaluating the
   delimiter. Cannot be used with RegexMatch.
-* **IgnoreCase**: Forces case-insensitive matching, even if the -cSplit
+- **IgnoreCase**: Forces case-insensitive matching, even if the -cSplit
   operator is specified.
 
 The RegexMatch options are:
 
-* **RegexMatch**: Use regular expression matching to evaluate the
+- **RegexMatch**: Use regular expression matching to evaluate the
   delimiter. This is the default behavior. Cannot be used with
   SimpleMatch.
-* **IgnoreCase**: Forces case-insensitive matching, even if the -cSplit
+- **IgnoreCase**: Forces case-insensitive matching, even if the -cSplit
   operator is specified.
-* **CultureInvariant**: Ignores cultural differences in language
+- **CultureInvariant**: Ignores cultural differences in language
   when evaluting the delimiter. Valid only with RegexMatch.
-* IgnorePatternWhitespace: Ignores unescaped whitespace and
+- IgnorePatternWhitespace: Ignores unescaped whitespace and
   comments marked with the number sign (#). Valid only with
   RegexMatch.
-* **Multiline**: Multiline mode recognizes the start and end of lines
-  and strings. Valid only with RegexMatch. Singleline is the default.
-* **Singleline**: Singleline mode recognizes only the start and end of
-  strings. Valid only with RegexMatch. Singleline is the default.
-* **ExplicitCapture**: Ignores non-named match groups so that only
+- **Multiline**: Multiline mode forces `^` and `$` to match the beginning
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
+- **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
+> **Singleline** and **Multiline** cannot be used together with the options
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
```

Comment:
If you can use both options is this statement needed?

Reply:
removed whole block.